code/modules/clothing/spacesuits/spacesuits.dm

```diff
@@ -48,7 +48,7 @@
 
 /obj/item/clothing/suit/space
 	name = "Space suit"
-	desc = "A suit that protects against low pressure environments. \"NORTHERN STAR\" is written in large block letters on the back."
+	desc = "A suit that protects against low pressure environments. \"NSS VORE\" is written in large block letters on the back."
```

Comment:
Our current one says NSS VORE, so this is different. Just checking to make sure that's what you want.